### PR TITLE
fix typo in font name: PPNeueBit-Bold

### PR DESCRIPTION
**Pull request by lukey-luke**
Fixes https://github.com/VerdanceTeam/verdance-site/issues/36

### CODE REVIEW

File: src/assets/styles/_foundation.scss [hunk 1]
```diff
@@ -246,7 +246,6 @@ $mobile-margin: 16px;
 
 .strikethrough {
   position: relative;
-  display: inline-block;
```
Comment (lukey-luke) [inline src/assets/styles/_foundation.scss:249]: Note to reviewer, updated so this class works w/ new font to look more like current production site

`'PPNeueBit-Bold'` with `display: inline-block;`
Screenshot 2024-08-05 at 3 57 05 PM

`'PPNeueBit-Bold'` with default `display` property
Screenshot 2024-08-05 at 3 57 09 PM

**Review by etanb — APPROVED** [review]

**Comment by amprokop** [comment]
Awesome! Thank you for looking into it. Can we bump up the font size?

It looks kind of comically small next to our headline font to my eyes

image

**Comment by amprokop** [comment]
image
current font size for comparison

**Comment by lukey-luke** [comment]
Screenshot 2024-08-07 at 10 42 49 AM

adjusted font sizes to be close to Tahoma font used in prod

**Review by amprokop — APPROVED** [review]
Looks good, I think we might want to switch out that font in general,  but this works for now.

**Pull request merged**